app/frontend/src/features/messages/Messages.tsx

```diff
@@ -0,0 +1,23 @@
+import { TabContext, TabPanel } from "@material-ui/lab";
+import * as React from "react";
+import PageTitle from "../../components/PageTitle";
+import TabBar from "./TabBar";
+import { labels } from "./constants";
+
+export default function Messages() {
+  const [value, setValue] = React.useState<keyof typeof labels>("TAB_ALL");
+  return (
+    <>
+      <PageTitle>Messages</PageTitle>
+      <TabContext value={value}>
+        <TabBar value={value} setValue={setValue} labels={labels} />
```

Comment:
This needs `scrollButton="auto"` or something like that, I can't scroll when doing mobile preview on Edge.

Reply:
Fixed